scripts/getPricesDAlayer.ts

```diff
@@ -0,0 +1,206 @@
+import { HardhatRuntimeEnvironment } from "hardhat/types";
```

Comment:
The name for prices is Anchor prices - use this as file name and inside the file